etc/kayobe/networks.yml

```diff
@@ -35,7 +35,9 @@ internal_net_name: internal
 # containing one item, external_net_name.
 #external_net_names:
 external_net_names:
-  - ilab
```

Comment:
Are we really removing access to ilab?

Reply:
Not really, but as it stands having it in the list will give the compute nodes an IP on ilab, which we don't really want. What we want is for them to use the raw tagged interface, which the overcloud provisioning network represents. Once we start using the admin network for SSH, we can swap oc_provision for eod_mgmt, which really represents the raw, tagged network.